docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -161,7 +161,7 @@ of `DataObject`. Adding this extension to children of the base class will have u
 
 You can use the `owns` static private property on a DataObject to specify which relationships are ownership relationships. The `owns` property should be defined on the _owner_ DataObject.
 
-For example, let's say you have a `MyPage` page type that displays banners containing an image. Each `MyPage` owns many `Banners`, which in turn owns an `Image`.
+For example, let's say you have a `MyPage` page type that displays banners containing an image. Each `MyPage` owns many `Banners`, which in turn own an `Image`.
```

Comment:
`owns` is right here.

Reply:
sorry - as in, it *should* be "owns"

Reply:
Oh - sorry for the misunderstanding - I'll change it back right, then